src/components/cards/cards.jsx

```diff
@@ -85,32 +85,46 @@ const CardHeader = ({onCloseCards, onShrinkExpandCards, onShowAll, totalSteps, s
     </div>
 );
 
-// Video step needs to know if the card is being dragged to cover the video
-// so that the mouseup is not swallowed by the iframe.
-const VideoStep = ({video, dragging}) => (
-    <div className={styles.stepVideo}>
-        {dragging ? (
-            <div className={styles.videoCover} />
-        ) : null}
-        <iframe
-            allowFullScreen
-            allowTransparency="true"
-            frameBorder="0"
-            height="257"
-            scrolling="no"
-            src={`https://fast.wistia.net/embed/iframe/${video}?seo=false&videoFoam=true`}
-            title="📹"
-            width="466"
-        />
-        <script
-            async
-            src="https://fast.wistia.net/assets/external/E-v1.js"
-        />
-    </div>
-);
+class VideoStep extends React.Component {
+    componentDidMount () {
+        const script = document.createElement('script');
```

Comment:
One concern here: this adds script elements to the page, but never removes them.

Maybe you could do something like:

Call `script.setAttribute('id', 'wistia-video-content')` here, and then in `componentWillUnmount` you could call

```
var wistiaScript = document.getElementById('wistia-video-content');
wistiaScript.parentNode.removeChild(wistiaScript);
```

Reply:
@benjiwheeler Ah great idea, thanks!  I will try that.